HTTPServer.test/ServerTests.cs

```diff
@@ -7,11 +7,34 @@ namespace HTTPServer.test
 {
     public class ServerUnitTests
     {
+        Server server = new Server();
+
         [Fact]
         public void ServerCanStart()
         {
-            var server = new Server();
-            Assert.True(server.Start() && server.running == false);
+            Assert.True(server.Start() && server.Running);
+        }
+
+        [Fact]
+        public void ServerCanStop()
+        {
+            server.Start();
+            Assert.True(server.Stop() && !server.Running);
+        }
+
+        [Fact]
+        public void ServerCanRespond200()
+        {
+            FileConnection file = new FileConnection();
```

Comment:
You should generally avoid reading/writing files/DBs/whatever in unit tests.  It slows things down, prevents any kind of parallelism, and puts your tests at risk of failing for some reason other than your code under test.